debbuild.spec

```diff
@@ -33,11 +33,13 @@ Requires:       lsb-release
 Requires:       xz-utils
 Recommends:     dpkg-sig
 Suggests:       rpm
+Recommends:     ruby
```

Comment:
Can you move this above the `Suggests` line so it's with other `Recommends`?

Reply:
I've changed both of these to `Suggests` instead since ruby is only needed if you're packaging a gem.

Reply:
LGTM!